packages/core/src/lib/features/FeatureManager.ts

```diff
@@ -81,16 +90,18 @@ export class FeatureManager {
     });
 
     // Adds properties to application
-    feature.applicationClass.imports.forEach((imp) => {
-      this.applicationClass.imports.add(imp);
-    });
-
-    feature.applicationClass.variables.forEach((imp) => {
-      this.applicationClass.variables.push(imp);
-    });
-
-    if (feature.applicationClass.onCreate) {
-      this.applicationClass.onCreate.push(feature.applicationClass.onCreate);
+    if (feature.applicationClass !== undefined) {
```

Comment:
Why does this need to be checked? I thought `applicationClass` can't be null/undefined.